songs/109_Sista_punschvisan.md

```diff
@@ -15,4 +15,6 @@ Och finns det ändå något kvar
 får det va' till sämre dar. ://
 
 > \* Ett vanligt symptom av sektionspatriotism är att man vrålar sin sektions
-> namn i mellanrummet: "båd' utanpå och i .. IN!!!"
+> namn i mellanrummet: "båd' utanpå och i... IT!!!"
+> Tidigare vrålade sektionen "IN", efter IN-sektionen.
+> Flera andra sektionen vrålar också "igen", möjligen efter högljudda "IN".
```

Comment:
This change does make sense. But since `IN` also makes sense as it sounds like "igen" which also fits into the context of the song, and (as you point out in the comments) may other chapters also sing "igen" (possibly due to influence from us).

Is it worth considering keeping it as it is, with a comment explaining that the chapter used to be called IN, and a comment explaining that some chapters sing "sin sektions namn i mellanrummet"?

Though this would of course depend on what is the common shout-out in the chapter today

Reply:
Maybe it could be worded in some other way, but I feel the history is important. It could be added to the new notes attribute as well after that has been implemented.